.sass-lint.yml

```diff
@@ -7,6 +7,8 @@ files:
     - 'x-pack/legacy/plugins/rollup/**/*.s+(a|c)ss'
     - 'x-pack/legacy/plugins/security/**/*.s+(a|c)ss'
     - 'x-pack/legacy/plugins/canvas/**/*.s+(a|c)ss'
+  ignore:
+    - 'x-pack/legacy/plugins/canvas/external_runtime/**/*.s+(a|c)ss'
```

Comment:
*operations*: I had to add an ignore here because the runtime is using CSS Modules, which confuses the linter.